eks-distro-base/Makefile

```diff
@@ -776,6 +780,9 @@ update-base-image-other-repos:
 .PHONY: all
 all: release
 
+.PHONY: rebuild-minimal-images
```

Comment:
let's add docstring-like comments to these to document what they're for, one sentance is fine